### PR TITLE
chore(release): Preparing for 3.9.1 release

### PR DESCRIPTION
**Pull request by The-inside-man**

### CODE REVIEW

**Review by jaeopt — CHANGES_REQUESTED** [review]
master has optConfigV2 fix that is not released yet. We need release this fix from v3.9.0

**Comment by The-inside-man** [comment]
> master has optConfigV2 fix that is not released yet. We need release this fix from v3.9.0

I was told to do this PR, then create a new release branch from v3.9.0 and cherrypick this commit and the fix commit once ready. FSC has a few failures so need to make some changes first.

**Comment by The-inside-man** [comment]
Once merged in I will create a new branch of release V3.9.0 commit and cherry pick this and the other fix commit into the new release branch which will be what we release for the Python SDK.

File: optimizely/event/user_event_factory.py [hunk 1]
```diff
@@ -45,12 +45,12 @@ def create_impression_event(
         if not activated_experiment and rule_type is not enums.DecisionSources.ROLLOUT:
             return None
 
-        variation, experiment_key = None, None
+        variation, experiment_id = None, None
```
Comment (msohailhussain) [inline optimizely/event/user_event_factory.py:48]: update license header

**Pull request closed without merging**